tests/countries/test_singapore.py

```diff
@@ -149,11 +149,12 @@ def test_2023(self):
         self.assertIn(date(2023, 6, 2), self.holidays)
         self.assertIn(date(2023, 6, 29), self.holidays)
         self.assertIn(date(2023, 8, 9), self.holidays)
+        self.assertIn(date(2023, 9, 1), self.holidays)
         self.assertIn(date(2023, 11, 12), self.holidays)
         self.assertIn(date(2023, 11, 13), self.holidays)
         self.assertIn(date(2023, 12, 25), self.holidays)
         # 2023: total holidays (11 + 3 falling on a Sunday)
```

Comment:
It seems the comment needs to be updated as well.

Reply:
Good catch! Done.